client/js/views/GameView.vue

```diff
@@ -268,13 +268,13 @@
       <!-- History -->
       <div v-if="$vuetify.breakpoint.smAndUp" class="history-container">
         <div id="field-right">
-          <div id="history" class="rounded d-flex flex-column justify-start">
-            <h3>History</h3>
```

Comment:
Would it be better to make a new scoped class for this? Something like `rounded-20` as a global class so you can use it anywhere? I'm not saying we should, but since we're using Vuetify, it feels more Vuetify-like. I'm fine leaving it though. Would love to move these sorts of global things out to CSS variables at some point so we can keep a manifest of logical defaults higher in the stack.

Reply:
Yeah I agree and think it would be a great candidate for a variable.